cant use css! inside a <Link> component from yew

im trying to customize the <Link> component from yew

this is an erronoues example
```rust
use stylist::yew::styled_component;
use yew::prelude::*;
use yew_router::prelude::*;


#[styled_component(Demo)]
fn demo_comp() -> Html {
    html! {
        <Link<AppRoutes> classes={
            css!(r#"
                    -webkit-appearance: button;
                    -moz-appearance: button;
                    appearance: button;
                    margin-top: 10px;
                    padding: 10px;
                    text-decoration: none;

                    position: absolute;

                    border-radius: 25px;
                    color: greenyellow;
                    background-color: #333;
                    font-family: monaco;
                    font-weight: bold;
                "#)
        } to={AppRoutes::HomeRoute}>{ "go back where you came from" }</Link<AppRoutes>>
    }
}
```

im trying to make this work, how do i do it?

here is the error
```shell
error[E0277]: the trait bound `StyleSource<'_>: IntoPropValue<Classes>` is not satisfied
  --> src/components/about.rs:9:26
   |
9  |         <Link<AppRoutes> classes={
   |                          ^^^^^^^ the trait `IntoPropValue<Classes>` is not implemented for `StyleSource<'_>`
   |
note: required by a bound in `LinkPropsBuilder::<R, LinkPropsBuilderStep_missing_required_prop_to, Q>::classes`
```

is there a possible way to customize <Link<T>> component with this crate ?

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,5 +1,11 @@
 # Changelog
 
+## v0.10.1
+
+### Other Changes:
+- Added an impl of `IntoPropValue<Classes>` for `Style` and `StyleSource` when
+  the `yew_integration` feature is active.
+
 ## v0.10.0
 
 ### Breaking Changes:

--- a/README.md
+++ b/README.md
@@ -13,7 +13,7 @@ This is a fork of [css-in-rust](https://github.com/lukidoescode/css-in-rust).
 Add the following to your `Cargo.toml`:
 
 ```toml
-stylist = "0.9"
+stylist = "0.10"
 ```
 
 ## Usage

--- a/packages/stylist-core/src/ast/selector.rs
+++ b/packages/stylist-core/src/ast/selector.rs
@@ -27,7 +27,7 @@ impl ToStyleStr for Selector {
             if joined_s.contains('&') || joined_s.contains(":root") {
                 w.push_str(
                     &joined_s
-                        .replace("&", scoped_class.as_str())
+                        .replace('&', scoped_class.as_str())
                         .replace(":root", scoped_class.as_str()),
                 );
             } else {
@@ -43,7 +43,7 @@ impl ToStyleStr for Selector {
 
         // For global styles, if it contains &, it will be replaced with html.
         } else if joined_s.contains('&') {
-            w.push_str(&joined_s.replace("&", ":root"));
+            w.push_str(&joined_s.replace('&', ":root"));
         // For other styles, it will be written as is.
         } else {
             w.push_str(&joined_s);

--- a/packages/stylist/src/yew/mod.rs
+++ b/packages/stylist/src/yew/mod.rs
@@ -115,6 +115,18 @@ impl From<StyleSource<'_>> for Classes {
     }
 }
 
+impl IntoPropValue<Classes> for Style {
+    fn into_prop_value(self) -> Classes {
+        self.into()
+    }
+}
+
+impl IntoPropValue<Classes> for StyleSource<'_> {
+    fn into_prop_value(self) -> Classes {
+        self.into()
+    }
+}
+
 impl IntoPropValue<StyleSource<'static>> for Sheet {
     fn into_prop_value(self) -> StyleSource<'static> {
         self.into()

